ReferenceState.pyx

```diff
@@ -14,7 +14,8 @@ from scipy.integrate import odeint
 include 'parameters.pxi'
 
 cdef extern from "thermodynamic_functions.h":
-    double qt_from_pv(double p0, double pv)
+    inline double qt_from_pv(double p0, double pv)
+    inline double thetali_c(double p0, double T, double qt, double ql, double qi, double L)
```

Comment:
I think we can remove all the thetali part in this file, but please double check to make sure.